python/pacemaker/_cts/tests/reattach.py

```diff
@@ -89,23 +130,22 @@ def teardown(self, node):
         start(node)
 
         if not self._is_managed(node):
-            self._logger.log("Attempting to re-enable resource management on %s" % node)
             self._set_managed(node)
-            self._cm.cluster_stable()
 
-            if not self._is_managed(node):
-                self._logger.log("Could not re-enable resource management")
-                return self.failure("Could not re-establish resource management")
+        (rc, _) = self._enable_incompatible_rscs(node)
+        if rc != ExitStatus.OK:
+            return self.failure("Couldn't modify CIB to re-enable incompatible resources")
+
+        if not self._cm.cluster_stable():
+            return self.failure("Cluster did not stabilize after teardown")
+        if not self._is_managed(node):
+            return self.failure("Could not re-enable resource management")
 
         return self.success()
 
     def can_run_now(self, node):
         """ Return True if we can meaningfully run right now """
 
-        if self._find_ocfs2_resources(node):
-            self._logger.log("Detach/Reattach scenarios are not possible with OCFS2 services present")
-            return False
-
         return True
```

Comment:
This is the only spot where can_run_now could ever return False, so it can be removed entirely (as can the only base class's version, and the only spot it's ever called).